domenetjenester/behandling-prosessering/src/main/java/no/nav/ung/sak/behandling/prosessering/task/HentInntektHendelserTask.java

```diff
@@ -0,0 +1,172 @@
+import no.nav.k9.prosesstask.api.ProsessTask;
+import no.nav.k9.prosesstask.api.ProsessTaskData;
+import no.nav.k9.prosesstask.api.ProsessTaskGruppe;
+import no.nav.k9.prosesstask.api.ProsessTaskHandler;
+import no.nav.k9.prosesstask.api.ProsessTaskTjeneste;
+import no.nav.ung.kodeverk.behandling.aksjonspunkt.AksjonspunktDefinisjon;
+import no.nav.ung.kodeverk.behandling.aksjonspunkt.AksjonspunktStatus;
+import no.nav.ung.kodeverk.behandling.aksjonspunkt.Venteårsak;
+import no.nav.ung.sak.behandling.FagsakTjeneste;
+import no.nav.ung.sak.behandlingslager.behandling.Behandling;
+import no.nav.ung.sak.behandlingslager.behandling.repository.BehandlingRepository;
+import no.nav.ung.sak.behandlingslager.fagsak.Fagsak;
+import no.nav.ung.sak.domene.registerinnhenting.InntektAbonnentTjeneste;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+
+@ApplicationScoped
+@ProsessTask(value = HentInntektHendelserTask.TASKTYPE)
+public class HentInntektHendelserTask implements ProsessTaskHandler {
+
+    public static final String TASKTYPE = "registerinnhenting.hentInntektHendelser";
+    public static final String SEKVENSNUMMER_KEY = "sekvensnummer";
+
+    private static final Logger log = LoggerFactory.getLogger(HentInntektHendelserTask.class);
+
+    private InntektAbonnentTjeneste inntektAbonnentTjeneste;
+    private FagsakTjeneste fagsakTjeneste;
+    private BehandlingRepository behandlingRepository;
+    private ProsessTaskTjeneste prosessTaskTjeneste;
+    private Duration ventetidFørNesteKjøring;
+    private boolean oppfriskVedInkommendeInntektshendelseEnabled;
+
+    public HentInntektHendelserTask() {
+        // For CDI
+    }
+
+    @Inject
+    public HentInntektHendelserTask(InntektAbonnentTjeneste inntektAbonnentTjeneste,
+                                    FagsakTjeneste fagsakTjeneste,
+                                    BehandlingRepository behandlingRepository,
+                                    ProsessTaskTjeneste prosessTaskTjeneste,
+                                    @KonfigVerdi(value = "OPPFRISK_VED_INNKOMMENDE_INNTEKTSHENDELSE_ENABLED", required = false, defaultVerdi = "false") boolean oppfriskVedInkommendeInntektshendelseEnabled,
+                                    @KonfigVerdi(value = "HENT_INNTEKT_HENDElSER_INTERVALL", required = false, defaultVerdi = "PT1M") String ventetidFørNesteKjøring){
+        this.inntektAbonnentTjeneste = inntektAbonnentTjeneste;
+        this.fagsakTjeneste = fagsakTjeneste;
+        this.behandlingRepository = behandlingRepository;
+        this.prosessTaskTjeneste = prosessTaskTjeneste;
+        this.oppfriskVedInkommendeInntektshendelseEnabled = oppfriskVedInkommendeInntektshendelseEnabled;
+        this.ventetidFørNesteKjøring = Duration.parse(ventetidFørNesteKjøring);
+    }
+
+
+    @Override
+    public void doTask(ProsessTaskData prosessTaskData) {
+
```

Comment:
Jeg tenker vi bør unngå linjeskift nesten alle steder inne i metoder siden det begrenser hvor mye kode som er leselig pr skjermareal. 

Det finnes helt sikkert steder hvor det gir mening å legge linjeskift for å indikere at det er en forskjell over/under, vurder alltid i slike tilfeller om det  det er bedre å dra ut en metode. Fordel med metode er at den kan navngis og dermed kommunisere intensjonen